parsec/core/gui/instance_widget.py

```diff
@@ -47,6 +47,24 @@ async def _do_run_core(config, device, qt_on_ready):
                 await trio.sleep_forever()
 
 
+def check_macfuse_version() -> bool:
+    import os
+
+    macfuse_plist_path = "/Library/Filesystems/macfuse.fs/Contents/Info.plist"
+    if not os.path.exists(macfuse_plist_path):
+        # osxfuse installed, not macfuse
```

Comment:
osxfuse and macfuse are not the same thing ???? O_o

(maybe add a small comment about this, I guess one is a legacy version of the other 🤔 )